packages/dev/addons/src/lottie/lottie/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./animationParser";
+export * from "./parsedTypes";
+export * from "./rawTypes";

```

Comment:
Does it really make sense to export everything from index? I usually think if the index file ideally as being the "public" API. If some of this exported stuff is only used internally, maybe it should not be exported from index?